modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -938,12 +938,19 @@ export default class I3SConverter {
 
           if (this.generateTextures) {
             formats.push({name: '1', format: 'ktx2'});
-            const ktx2TextureData = encode(texture.image, KTX2BasisWriterWorker, {
-              ...KTX2BasisWriterWorker.options,
-              source: this.workerSource.ktx2,
-              reuseWorkers: true,
-              _nodeWorkers: true
-            });
+            const copyArrayBuffer = new ArrayBuffer(texture.image.data.length);
```

Comment:
you can use `arrayBuffer.slice()` to create a copy. This is used e.g. in test cases.